components/benefit_sponsors/app/models/effective/datatables/benefit_sponsors_employer_datatable.rb

```diff
@@ -270,6 +244,95 @@ def nested_filter_definition
       def authorized?(current_user, _controller, _action, _resource)
         current_user.present? && current_user.has_hbx_staff_role?
       end
+
+      def collection
+        return @collection if defined? @collection
+
+        benefit_sponsorships ||= BenefitSponsors::BenefitSponsorships::BenefitSponsorship.unscoped
+        benefit_sponsorships = filter_by_employers(benefit_sponsorships)
+        benefit_sponsorships = filter_by_enrolling(benefit_sponsorships)
+        benefit_sponsorships = filter_by_enrolled(benefit_sponsorships)
+        benefit_sponsorships = filter_by_employer_attestations(benefit_sponsorships)
+        benefit_sponsorships = filter_by_upcoming_dates(benefit_sponsorships)
+        benefit_sponsorships = filter_by_attestations(benefit_sponsorships)
+
+        @collection = benefit_sponsorships
+      end
+
+      private
+
+      def filter_by_employers(benefit_sponsorships)
+        return benefit_sponsorships unless attributes[:employers].present? && !['all'].include?(attributes[:employers])
+
+        call_safe_method(benefit_sponsorships, attributes[:employers]) if ALLOWED_METHODS.include?(attributes[:employers])
```

Comment:
Looks like you already check if the method is allowed in  `call_safe_method`.
Why you check `ALLOWED_METHODS.include?(attributes[:employers])` here?